pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
Part of what I said was not true. Our client library does have dependency on the API, e.g., https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/scale.go#L21. However, the dependency is on a specific version of a group, while the changes in this PR will create a dependency on all versions (<group>/latest imports all versions), so the old way still has a smaller import graph.